dmutils/__init__.py

```diff
@@ -2,4 +2,4 @@
 from .flask_init import init_app, init_manager
 
 
-__version__ = '49.0.0'
+__version__ = '50.0.0'
```

Comment:
What should go in the CHANGELOG entry for this?

Reply:
"Look out!"